scripts/docker/Dockerfile

```diff
@@ -12,19 +13,25 @@ RUN nrfutil toolchain-manager install --toolchain-bundle-id $VERSION \
     && nrfutil toolchain-manager env --as-script > /opt/toolchain-env.sh \
     && rm -rf /opt/ncs/downloads
 
+# Prepare JLink to be installed when user accepts license agreement
+COPY --chmod=0555 jlink/ /jlink
+RUN wget --post-data="accept_license_agreement=accepted&submit=Download+software" -O /jlink/JLink_Linux.deb \
+    https://www.segger.com/downloads/jlink/JLink_Linux_V794e_x86_64.deb
+
+# Add interactive-setup.sh which is called at the beginning of bash session, only if session is interactive
+# interactive-setup.sh is saved as bashrc file in root's home folder (/opt)
+COPY --chmod=0555 interactive-setup.sh /opt/.bashrc
```

Comment:
I would avoid adding two different paths (interactive and non-interactive) if possible. With one execution path container would work similarly in all use cases (e.g. CI or locally). Also there is less scripts to maintain and it is easier to document/understand.

Reply:
I agree, but I failed one-file approach. When non-interactive script is executed, we cannot determine if session is interactive or not

Reply:
I don't get why detection of interactivity is important. So IMO it should not start asking question in case of interactive. Instead just behave similarly as in non-interactive case (just rely env ACCEPT_JLINK_LICENSE). So I think line 23 is not needed.